### ISSUE TITLE
get kappa vector

### ISSUE THREAD

**Issue by icupeiro** [MassimoCimmino/pygfunction#66]
Hi Massimo,

Correct me if I missed it, but I could not find any means to extract the kappa vector with the dimensional thermal response factors in pygfunction.
I did a small tweak on my branch of pygfunction to extract the kappa vector. In the load_aggregation.py file, within the class ClaessonJaved I added the following function

```
    def get_kappa(self):

        return self.dg
```

It is a small addition, but perhaps can be useful in the code. If you are interested I can add it to the classes MLAA and Liu and make a pull request.

Best,
Iago

**Comment by MassimoCimmino** [comment]
Hello @icupeiro.

You are right, that could indeed be useful.

Since "kappa" is not necessarily an established term for this vector, I suggest : `get_thermal_response_factor_increment(self)`. Would you also be willing to complete the docstring for the class method?

This seems straightforward so I will add it to the `1.1.2` milestone. I am aiming to push the update on Pypi in a week or so.

**Comment by icupeiro** [comment]
Consider it done

It may be more useful to re-shape the matrix to an array, for example as:

`return self.dg.reshape(len(self._time))`

Let me know your preference

**Comment by MassimoCimmino** [comment]
After some thought I recalled that the thermal response factor increments are only used for the `ClaessonJaved` method (It is the only method where the load history cells have fixed times). I removed the method from `MLAA` and `Liu` since `dg` is not used in these cases.

For the shape of the array, a simple fix was to return `self.dg.flatten()`. This should provide the expected behaviour, returning a `(nSources, nSources, Nt)` array if `nSources > 1` and a 1d array of length `Nt` if `nSources == 1`.
